CHANGES.md

```diff
@@ -1,5 +1,12 @@
 # Changes
 
+### v0.4.4
+
+* Remove `GroupMetricSet` in favour of a `create_group_metric_set` method
+* Add `to_dict()` and `__repr__()` methods to `GroupMetricResult`
+* Add basic support for multiple sensitive features
```

Comment:
Thank you! I thought I had written that somehow...

Reply:
However, I think that your build fix needs to go in first....